services/src/main/java/org/keycloak/forms/login/freemarker/model/WebAuthnAuthenticatorsBean.java

```diff
@@ -34,22 +39,35 @@ public WebAuthnAuthenticatorsBean(KeycloakSession session, RealmModel realm, Use
                 .map(WebAuthnCredentialModel::createFromCredentialModel)
                 .map(webAuthnCredential -> {
                     String credentialId = Base64Url.encodeBase64ToBase64Url(webAuthnCredential.getWebAuthnCredentialData().getCredentialId());
-                    String label = (webAuthnCredential.getUserLabel()==null || webAuthnCredential.getUserLabel().isEmpty()) ? "label missing" : webAuthnCredential.getUserLabel();
-                    return new WebAuthnAuthenticatorBean(credentialId, label);
+                    String label = (webAuthnCredential.getUserLabel() == null || webAuthnCredential.getUserLabel().isEmpty()) ? "label missing" : webAuthnCredential.getUserLabel();
+                    String createdAt = getDateTimeFromMillis(webAuthnCredential.getCreatedDate());
+                    return new WebAuthnAuthenticatorBean(credentialId, label, createdAt);
                 }).collect(Collectors.toList());
     }
 
+    private String getDateTimeFromMillis(long millis) {
+        return getDateTimeFromMillis(millis, "dd-MM-yyyy HH:mm");
```

Comment:
Probably, it'd be better to define the date format somewhere in properties file in order to have possibility to change it by admin.

Reply:
I would use the same stuff, which some other places in Keycloak user pages are using? For example, in account console in the "Device activity" page, there are some dates displayed.

Reply:
Unfortunately, AFAIK the date is not displayed anywhere in login theme. There's some JS/TS util class for account console, but we are not able to use it here. Therefore, we need to create custom time formatter.